assets/js/services/helpers.test.js

```diff
@@ -0,0 +1,75 @@
+import {
+  buildImageURL,
+  escapeDoubleQuotes,
+  formatDate,
+  getClassFromIngestSheetStatus
+} from "./helpers";
+
+describe("Build Image URL function", () => {
+  it("should return URL", () => {
```

Comment:
Let's make this a little more specific.  How about something like "should return a URL containing the correct IIIF params..."

Reply:
Cool, I'll make this change!